worker_selector.py

```diff
@@ -53,23 +66,49 @@
     dynamic_game = True
     infinite_boost_odds = 0
     host = "127.0.0.1"
+
     if len(sys.argv) > 1:
         host = sys.argv[1]
         if host != "127.0.0.1" and host != "localhost":
             local = False
+
     if len(sys.argv) > 2:
         name = sys.argv[2]
-    # if len(sys.argv) > 3 and not dynamic_game:
```

Comment:
Hope it was okay that I removed this, it wasn't used, and a constant was set elsewhere.